gluoncv/auto/estimators/ssd/default.py

```diff
@@ -0,0 +1,88 @@
+from sacred import Experiment, Ingredient
+
+from ..common import logging
+
+ssd = Ingredient('ssd')
+train = Ingredient('train')
+validation = Ingredient('validation')
+
+
+@ssd.config
+def ssd_default():
+    # Backbone network.
+    backbone = 'vgg16_atrous'  # base feature network
+
+    # Use synchronize BN across devices.
+    syncbn = False
+
+    # Input data shape, use 300, 512.
+    data_shape = 300
+
+    # Whether to use automatic mixed precision
+    amp = False
+
+
+@train.config
+def train_cfg():
+    # Batch size during training
+    batch_size = 32
+    # starting epoch
+    start_epoch = 0
+    # total epoch for training
+    epochs = 240
+
+    # Solver
+    # ------
+    # Learning rate.
+    lr = 0.001
+    # Decay rate of learning rate.
+    lr_decay = 0.1
+    # Epochs at which learning rate decays
+    lr_decay_epoch = (160, 200)
+    # Learning rate scheduler mode. options are step, poly and cosine
+    lr_mode = 'step'
+    # Momentum
+    momentum = 0.9
+    # Weight decay
+    wd = 5e-4
+
+    # Misc
+    # ----
+    # log interval in terms of iterations
+    log_interval = 100
+    seed = 233
+
+    # Use DALI for data loading and data preprocessing in training.
+    # Currently supports only COCO.
+    dali = False
+
+
+@validation.config
+def valid_cfg():
+    # Epoch interval for validation
+    val_interval = 1
+
+
+ex = Experiment('ssd_default', ingredients=[logging, train, validation, ssd])
+
+
+@ex.config
+def default_configs():
```

Comment:
as discussed offline, we can use `get_ssd` function and expose more options to customize the network structure here